.travis.yml

```diff
@@ -26,6 +26,7 @@ cache:
   - "$HOME/.m2"
   - "$HOME/apache-maven-3.5.0"
   - "$TRAVIS_BUILD_DIR/com.ibm.wala.core.testdata/ocaml/ocamljava-2.0-alpha1/lib"
+  - "$TRAVIS_BUILD_DIR/build/p2asmaven"
```

Comment:
Swap with preceding line to keep this list in alphabetical order.

Reply:
Fixed